syft/frameworks/torch/dp/pate.py

```diff
@@ -241,9 +254,6 @@ def perform_analysis(teacher_preds, indices, noise_eps, delta=1e-5, moments=8, b
 
     eps_list_nm = (total_log_mgf_nm - math.log(delta)) / l_list
 
-    # print("Epsilons (Noisy Max): " + str(eps_list_nm))
-    # print("Smoothed sensitivities (Noisy Max): " + str(total_ss_nm / l_list))
```

Comment:
do you want to rm this?

Reply:
I think so. If i've understood it correctly, the original tensorflow code (https://github.com/tensorflow/privacy/blob/master/research/pate_2017/analysis.py#L272) used this before adding noise to estimate epsilon, which is why it prints "to get this epsilon, use this noise". This code is formulated slightly differently as we supply the noisy counts to the function. So even if the code wasn't commented-out it wouldn't make complete sense, as we've already added the noise